install.sh

```diff
@@ -270,6 +293,36 @@ libsecp256k1_install()
     popd
 }
 
+clightning_build ()
+{
+    ./configure \
+        --enable-developer \
+        --enable-experimental-features \
+        --prefix="${jm_root}"
+    $make
+}
+
+clightning_install ()
+{
+    # note: the normal tarball source is broken and must not be used according to:
+    # https://github.com/ElementsProject/lightning/issues/3900#issuecomment-668330656
+    # we use links like:
+    # https://github.com/ElementsProject/lightning/releases/download/v0.10.1/clightning-v0.10.1.zip
+    clightning_version='clightning-v0.10.1'
```

Comment:
v0.10.2 was released yesterday

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/7a5e4c3369966dda121ac01a09dd999071c8dc4a